manifests/2.13.0/opensearch-2.13.0.yml

```diff
@@ -0,0 +1,16 @@
+---
+schema-version: '1.1'
+build:
+  name: OpenSearch
+  version: 2.13.0
+ci:
+  image:
+    name: opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3
+    args: -e JAVA_HOME=/opt/java/openjdk-17
```

Comment:
Needs to be jdk 21: https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.12.0/opensearch-2.12.0.yml#L9

@prudhvigodithi @zelinh Can you look into this?

Reply:
Folks please check https://github.com/opensearch-project/opensearch-build/pull/4429

Reply:
It gets the JDK version from template manifest. https://github.com/opensearch-project/opensearch-build/blob/a31e67173f4eedb2abb5b2f285a8c9dce37592e6/manifests/templates/opensearch/2.x/manifest.yml#L9
Let me update it.

Reply:
Thank you!

Reply:
PR raised here. https://github.com/opensearch-project/opensearch-build/pull/4431